static/app/views/projectDetail/projectScoreCards/projectApdexScoreCard.tsx

```diff
@@ -14,199 +14,162 @@ import type {Organization} from 'sentry/types/organization';
 import {defined} from 'sentry/utils';
 import type {TableData} from 'sentry/utils/discover/discoverQuery';
 import {getPeriod} from 'sentry/utils/duration/getPeriod';
+import {useApiQuery} from 'sentry/utils/queryClient';
 import {getTermHelp, PerformanceTerm} from 'sentry/views/performance/data';
 
 import MissingPerformanceButtons from '../missingFeatureButtons/missingPerformanceButtons';
 
-type Props = DeprecatedAsyncComponent['props'] & {
+type Props = {
   isProjectStabilized: boolean;
   organization: Organization;
```

Comment:
nit: You could use `useOrganization` instead.

Reply:
Roger! There's going to be a second round of refactors for these, I can address then 👍🏻